packages/api/.env.template

```diff
@@ -66,6 +66,11 @@ AZURE_RESOURCE_TAG_NAMES=["resourceGroup"] # eg. ["resourceGroup","project","cus
 # To avoid rate limiting, azure estimations can be chunked by days
 AZURE_CONSUMPTION_CHUNKS_DAYS=0
 
+# Ali
+
+ALI_INCLUDE_ESTIMATES=true
+ALI_USE_BILLING_DATA=true
+
```

Comment:
All variable options added to `packages/common/src/Config.ts` file needs to be listed here